client/src/App.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import RatingsAndReviews from './RatingsAndReviews/RatingsAndReviews';
+import { Products } from './dummyData.js';
+
+class App extends React.Component {
```

Comment:
if you're importing { Component } on line 1, I don't think we need to  extend from React.Component, but I don't see the harm in having it.

Reply:
@Gledinburgh Yeah it auto-corrected to use React.component instead of Component for some reason lol

Reply:
@Gledinburgh Will change that!!